Clean/Specs/Keccak256.lean

```diff
@@ -5,7 +5,7 @@ namespace Specs.Keccak256
 
 open Bitwise (not64 rot_left64)
 
-def roundConstants : Vector ℕ 24 := #v[
+def roundConstants : Vector UInt64 24 := #v[
```

Comment:
I changed the round constants type to `UInt64`, because the size bound is needed in proofs, and it's easier to have it in the type than to prove every time

Reply:
Good idea, makes a lot of sense 👍